### PR TITLE
Feature/docker optimization

### PR DESCRIPTION
**Pull request by iluxonchik**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,56 +1,33 @@
...
-FROM node:20-alpine AS builder
-WORKDIR /app
+RUN npm ci
 
-# Install dependencies
-COPY package*.json ./
 COPY prisma ./prisma/
-RUN npm install
+COPY src ./src
+COPY public ./public
```
Comment (johnmarcou) [inline Dockerfile:11]: **issue:** We need to remove the `public` folder here, as it does not exist before building.

File: docker-compose.yml [hunk 2]
```diff
@@ -47,26 +34,10 @@ services:
...
+    restart: unless-stopped
 
 volumes:
   postgres_data:
-  pgadmin_data:
-  node_modules:
-  next-cache:
 
 networks:
   app_network:
```
Comment (johnmarcou) [inline docker-compose.yml:43]: **suggestion:** We could remove the custom network and let compose to use the default project one.

```suggestion
```

We can remove this section in both services:
```
    networks:
      - app_network
```

File: package.json
Comment (johnmarcou) [inline package.json:1]: **suggestion:** I think we do need to handle prism DB migration on start.

**Comment by iluxonchik** [comment]
Addressed both in new push

**Review by johnmarcou — APPROVED** [review]
LGTM!

**Pull request merged**